pandas/tests/arrays/period/test_arrow_compat.py

```diff
@@ -60,9 +57,6 @@ def test_arrow_array(data, freq):
     with pytest.raises(TypeError, match=msg):
         pa.array(periods, type="float64")
 
-    with pytest.raises(TypeError, match="different 'freq'"):
-        pa.array(periods, type=ArrowPeriodType("T"))
```

Comment:
Were had an xfail for `not pa_version_under10p1`, so essentially we were letting this xfail for _all_ our supported pyarrow versions. They fail because this last  check, and so I removed this failing step and then also removed the xfail (in any case, it is behaviour in pyarrow outside of our control, AFAIK)